test/Opdex.Platform.WebApi.Tests/Validation/SnapshotFilterParametersValidatorTests.cs

```diff
@@ -0,0 +1,160 @@
+using FluentValidation.TestHelper;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Queries;
+using Opdex.Platform.WebApi.Models.Requests;
+using Opdex.Platform.WebApi.Validation;
+using System;
+using Xunit;
+
+namespace Opdex.Platform.WebApi.Tests.Validation
+{
+    public class SnapshotFilterParametersValidatorTests
```

Comment:
Another thought for future tasks we should probably come back to these and assert the messages we want are returned.

Reply:
Added reason validation and created a backlog item for the existing validation tests